src/base/gtest/serialization_fixture.hh

```diff
@@ -98,9 +98,8 @@ class SerializationFixture : public ::testing::Test
     SetUp() override
     {
         // Create the directory
-        dirName = generateTempDirName();
-        [[maybe_unused]] int success = mkdir(dirName.c_str(), 0775);
-        assert(!(success == -1 && errno != EEXIST));
+        [[maybe_unused]] char* ret = mkdtemp(dirName.data());
```

Comment:
The reason for not use `dirName = /tmp/ + generateTempDirName()` here is because that it is possible that a file with the name returned by std::tmpnam is created by another process between the moment std::tmpnam returns and the moment this program attempts to use the returned name to create a file.